client/src/components/ProfileSettingsSkeleton/ProfileSettingsSkeleton.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState, SyntheticEvent } from 'react';
+import { Link, Switch, Route } from 'react-router-dom';
+import { useHistory } from 'react-router';
+import clsx from 'clsx';
+import MenuIcon from '@material-ui/icons/Menu';
+import { Box, Grid, Typography, Button, Menu, MenuItem } from '@material-ui/core';
+import links from './ProfleSettingsLInks';
+import useStyles from './useStyles';
+
+export default function ProfileSkeleton(): JSX.Element {
+  const classes = useStyles();
+  const { location } = useHistory();
+  const [element, setElement] = useState<(EventTarget & Element) | null>(null);
+  const open = Boolean(element);
+
+  const handleClick = (event: SyntheticEvent) => {
+    setElement(event.currentTarget);
+  };
+  const handleClose = () => {
+    setElement(null);
+  };
+
+  const isActive = (path: string): boolean => {
+    if (location.pathname === `/profile${path}`) return true;
+    return false;
+  };
+
+  return (
+    <Grid container className={classes.root}>
+      <Box component="nav" className={classes.nav}>
+        {links.map((link) => (
+          <Link className={classes.link} key={link.name} to={`/profile${link.path}`}>
+            <Typography align="left" className={clsx(classes.linkText, isActive(link.path) && classes.active)}>
+              {link.name}
+            </Typography>
+          </Link>
```

Comment:
The use of `clsx` here is very cool 💯

An even better alternative would be to combine [`Lists` from Material UI](https://mui.com/components/lists/#main-content) and `react-router-dom` in my opinion but this definitely does the job as well 👍🏼

Reply:
Thanks, I'll take a look at it.